File: sdkwork-ai-sdk-python/sdkwork_ai_sdk/models/chat_message.py
from dataclasses import dataclass
from typing import Optional, List, Dict, Any

@dataclass
class ChatMessage:
    """OpenAI标准聊天消息"""
    role: str = None
    content: Any = None
    name: str = None
    refusal: str = None
    audio: Audio = None
    annotations: List[Annotation] = None
    tool_calls: List[ToolCall] = None
    tool_call_id: str = None

File: sdkwork-ai-sdk-kotlin/src/main/kotlin/com/sdkwork/ai/ChatMessage.kt
package com.sdkwork.ai

data class ChatMessage(
    val role: String? = null,
    val content: Any? = null,
    val name: String? = null,
    val refusal: String? = null,
    val audio: Audio? = null,
    val annotations: List<Annotation>? = null,
    val toolCalls: List<ToolCall>? = null,
    val toolCallId: String? = null
)
